OpenUpSA/wazimap-ng-ui
# hiding the point category legend and only showing it in map download

## Description
hiding the point category legend and only showing it in map download

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-138

## How is it tested automatically?
* added a step in `__tests__/gui/point_data.feature` which checks if the point category legend is hidden and has `.visible-in-download` class. elements that have `.visible-in-download` class are shown in map download

## How should a reviewer test it locally
* select some categories from point mapper
* wait until the points are displayed
* confirm that point category legend is not shown
* download map
* confirm that the point category legend is visible in the downloaded image

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/140526353-68f3d17e-c54c-4c82-8629-71ce0bf730fc.png)


## Changelog

### Added

### Updated
* `src/js/map/point_data/point_data.js` to not remove `hidden` class from `.map-point-legend` element and add `visible-in-download` class to it
* `src/js/map/map_download.js` to remove class `hidden` from the elements that have `visible-in-download` class
* `__tests__/gui/point_data.feature` and `__tests__/gui/point_data/point_data.js` to test this

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
